pkg/kubelet/apis/config/validation/validation_test.go

```diff
@@ -85,8 +85,10 @@ func TestValidateKubeletConfiguration(t *testing.T) {
 		NodeLeaseDurationSeconds:    -1,
 		CPUCFSQuotaPeriod:           metav1.Duration{Duration: 0},
 		TopologyManagerPolicy:       "",
+		KubeReserved:				 map[string]string{"cpu": "-200m", "memory": "-200G"},
```

Comment:
I think this will most likely fail golint - we want it to align with the other values in the dict :)

Reply:
Run hack/update-gofmt.sh.